templates/pulsar.service.j2

```diff
@@ -9,10 +9,12 @@ UMask=022
 Type=simple
 User={{ __pulsar_user_name }}
 Group={{ __pulsar_user_group }}
-WorkingDirectory={{ pulsar_server_dir }}
+WorkingDirectory={{ pulsar_root }}
 TimeoutStartSec=30
 {% if pulsar_systemd_runner == "paste" %}
 ExecStart={{ pulsar_venv_dir }}/bin/python {{ pulsar_venv_dir }}/bin/paster serve {{ pulsar_config_dir }}/server.ini
+{% elif pulsar_systemd_runner == "webless" %}
+ExecStart={{ pulsar_venv_dir }}/bin/pulsar-main --ini_path {{ pulsar_config_dir }}/server.ini
```

Comment:
```suggestion
ExecStart={{ pulsar_venv_dir }}/bin/python {{ pulsar_venv_dir }}/bin/pulsar -c {{ pulsar_config_dir }} -m webless
```

I have uncommitted in-development changes to do the same and this is what I used - can you check if it works for you?

Reply:
Your modification doesn't work for me, instead 
https://github.com/galaxyproject/ansible-pulsar/pull/12/commits/5b9bcef09a23c2ca3b34402538ea6cbc89c29e4f yes